flutter/message.go

```diff
@@ -15,9 +15,9 @@ const (
 	TextUpdateStateMethod = "TextInputClient.updateEditingState"
 
 	// text
-	SetClientMethod       = "TextInput.setClient"
```

Comment:
should add a updateState method to textModel and handle all these methods in there.